src/styles/blocks/content.scss

```diff
@@ -0,0 +1,81 @@
+.clock {
+  position: relative;
+  width: 1em;
+  height: 1em;
+  box-sizing: border-box;
+  border: $padding-clock solid #000;
+  border-radius: 50%;
+  font-size: $size-clock;
+  text-align: center;
+
+  &__hand {
```

Comment:
do you need two dots? bc you have two `.clock__hand--time`

Reply:
understand. done